### PR TITLE
Logged header

### PR DESCRIPTION
**Pull request by pabloderivia**

### CODE REVIEW

File: .cache/08/4a36f35827ff05b103bc18ed5bd59f.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"id":"index.html","dependencies":[{"name":"./javascript/index.js","dynamic":true,"resolved":"/home/pablo/Documentos/myment-front/src/javascript/index.js","parent":"/home/pablo/Documentos/myment-front/src/index.html"}],"generated":{"html":"<!DOCTYPE html>\n<html lang=\"en\">\n<head>\n    <meta charset=\"UTF-8\">\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\n    <meta http-equiv=\"X-UA-Compatible\" content=\"ie=edge\">\n    <title>Document</title>\n    <script defer=\"\" src=\"/ac4aac0aa39d61dfd46dda1da9e043f0.js\"></script>\n</head>\n<body>\n\n    <div id=\"app\">\n    </div>\n\n</body>\n</html>"},"sourceMaps":null,"error":null,"hash":"9777be12c226581e3fe96674d2f63ebf","cacheData":{}}
```
Comment (ArcaDelDelirio) [inline .cache/08/4a36f35827ff05b103bc18ed5bd59f.json]: Okey the cache grow like a flower

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
 node_modules
+dist
+cache
```
Comment (ArcaDelDelirio) [inline .gitignore]: It's .cache and for don't check the dist, to must remove before and later add and commit

File: design/.~lock.Mock-Up Project.pdf# [hunk 3]
```diff
@@ -0,0 +1 @@
+,pablo,pablo-PC,14.07.2019 20:43,file:///home/pablo/.config/libreoffice/4;
```
Comment (ArcaDelDelirio) [inline design/.~lock.Mock-Up Project.pdf#]: Mmmm libre Office, you use some Linux dist

File: src/components/Button.vue [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  }
+
+</script>
+
+<style > 
+.Button {
+  color:white;
+  background: #454647;
+  background-image: -webkit-linear-gradient(top, #454647, #000000);
+  background-image: -moz-linear-gradient(top, #454647, #000000);
```
Comment (ArcaDelDelirio) [inline src/components/Button.vue]: One question ,Did you add manually all moz,WebKit... Things?

File: src/components/LoggedHeader.vue [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        <img src="../assets/M.png" alt="our coolest logo">
+        <newsBar class = "news"></newsBar>
+    </div>
+</template>
+
+<script>
+import newsBar from './NewsBar';
+export default {
+    components:{
+        'newsBar': newsBar
```
Comment (ArcaDelDelirio) [inline src/components/LoggedHeader.vue]: Not necessarily if you use the same name

File: src/components/LoggedHeader.vue [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+}
+</script>
+
+<style>
+.news{
+    height: 60px;
```
Comment (ArcaDelDelirio) [inline src/components/LoggedHeader.vue]: Pixels it's not responsive

File: src/components/NewsBar.vue [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    box-sizing: border-box;
+    border: double 3px grey;
+    box-shadow: inset 3px 3px 4px white, inset -3px -3px 4px white;
+}
+.marquee h6 {
+    display: inline-block;
+    padding-left: 100%;
+    animation: marquee 15s linear infinite;
+}
+@keyframes marquee {
```
Comment (ArcaDelDelirio) [inline src/components/NewsBar.vue]: Yeah!! Cool!

File: src/components/NewsBar.vue [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+</script>
+
+<style>
+
+.marquee {
+    display:flex;
+    align-items: center;
+    font-size: 20px;
+    width: 100%;
+    height: 50px;
```
Comment (ArcaDelDelirio) [inline src/components/NewsBar.vue]: Maybe we can use postcss to create some reference variables to follow the responsive way

**Comment by armandodecanha** [comment]
I think need to plan if the the web will be responsive on phones also. If that's the case, it will take a lot of time to do it. _"Flexbox tells an element how to take up space. Media Query tells you how the content is being viewed."_ So... we have to decide if we want that this web will be be used on a phone, on a pc, or both.

**Comment by ArcaDelDelirio** [comment]
The standard is make to mobile and later adapt to the pc. One feature of the app make photos more than picture on a place. For that the page may be first for mobile with an integrated 📷 for easy use, I think

**Comment by ArcaDelDelirio** [comment]
Please make review with comments or reply me for take the conversation organized :nerd_face:

**Pull request merged**